server/src/database/mod.rs

```diff
@@ -0,0 +1,35 @@
+use postgres::{Connection, TlsMode, Error};
+use std::borrow::Borrow;
+use std::{thread, time};
+use super::utils::get_env_variable;
+
+/// Try n-times to get a new connection to a Postgres database.
+fn try_connection(params: &String, tries: u8) -> Result<Connection, Error> {
+    match Connection::connect((*params).borrow(), TlsMode::None) {
+        Ok(conn) => Ok(conn),
+        Err(error) => {
+            match tries {
+                0 => Err(error),
+                _ => {
+                    println!("Could not connect to database... Retry in 10s!");
```

Comment:
Ok for now, but I suggest we eventually find a proper `logger` that we use in the entire backend.